src/Arduino_ConnectionHandler.h

```diff
@@ -113,34 +129,39 @@ class ConnectionHandler {
   #define NETWORK_CONNECTED GSM3_NetworkStatus_t::GPRS_READY
 #endif
 
+#if defined(ARDUINO_SAMD_MKRWAN1300) || defined(ARDUINO_SAMD_MKRWAN1310)
+  #include <MKRWAN.h>
+  #define BOARD_HAS_LORA
+#endif
+
 #if    defined(ARDUINO_ESP8266_ESP12)    \
-    || defined(ESP8266)                  \
-    || defined(ESP8266_ESP01)            \
-    || defined(ESP8266_ESP13)            \
-    || defined(ESP8266_GENERIC)          \
-    || defined(ESP8266_ESPRESSO_LITE_V1) \
-    || defined(ESP8266_ESPRESSO_LITE_V2) \
-    || defined(ESP8266_PHOENIX_V1)       \
-    || defined(ESP8266_PHOENIX_V2)       \
-    || defined(ESP8266_NODEMCU)          \
-    || defined(MOD_WIFI_ESP8266)         \
-    || defined(ESP8266_THING)            \
-    || defined(ESP8266_THING_DEV)        \
-    || defined(ESP8266_ESP210)           \
-    || defined(ESP8266_WEMOS_D1MINI)     \
-    || defined(ESP8266_WEMOS_D1MINIPRO)  \
-    || defined(ESP8266_WEMOS_D1MINILITE) \
-    || defined(ESP8266_WEMOS_D1R1)       \
-    || defined(ESP8266_ESP12)            \
-    || defined(WIFINFO)                  \
-    || defined(ESP8266_ARDUINO)          \
-    || defined(GEN4_IOD)                 \
-    || defined(ESP8266_OAK)              \
-    || defined(WIFIDUINO_ESP8266)        \
-    || defined(AMPERKA_WIFI_SLOT)        \
-    || defined(ESP8266_WIO_LINK)         \
-    || defined(ESP8266_ESPECTRO_CORE)
-  
+  || defined(ESP8266)                  \
```

Comment:
A formatting change should be avoided since it enlarges the diff in a meaningless way. Please always check your change by taking a look at `git diff` before committing it.